rundeckapp/grails-app/domain/rundeck/Option.groovy

```diff
@@ -367,6 +387,27 @@ public class Option implements Comparable{
         return opt
     }
 
+    Set<String> getValues(){
+        if(valuesList){
+            convertValuesList()
+        }
+        if(values && sortValues){
+            sortValuesList()
+        }
+        values
+    }
+
+    void sortValuesList(){
+        def numericValues = values.findAll { it.isNumber() }.collect {it.toDouble()}
+        if(numericValues.size()==values.size()){
+            def stringNumericList= values.findAll { it.isNumber() }.collect {new StringNumericSort(it, it.toDouble())}
+            stringNumericList.sort{ a,b -> a.numericValue <=> b.numericValue }
```

Comment:
the comparator logic should be moved to StringNumericSort where it could be unit tested